### PR TITLE
Adding clean up logic for AWS provisioning

### PR DESCRIPTION
**Pull request by ArvinthC3000**
Signed-off-by: Arvinth C <arvinth.chandrasekaran@progress.com>

### :nut_and_bolt: Description: What code changed, and why?
In this PR, Cleanup script for AWS provisioning is added

Commands available:
- --aws-deployment

Example:
`chef-automate cleanup --aws-deployment --force`

### :chains: Related Resources
Jira Ticket: https://chefio.atlassian.net/browse/MADROX-354

### :+1: Definition of Done
It's dev done and tested

### :athletic_shoe: How to Build and Test the Change

- Use following CLI package to create chef-autoomate utility
- Use the following 'HA Deployment' package to create the airgap-bundle for provisioning

CLI package: `arvinth/automate-cli/0.1.0/20221103103437`
Backend-deployment packages: `arvinth/automate-ha-deployment/0.1.0/20221102173230`

### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [x] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [x] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [x] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [x] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [x] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable

image

### CODE REVIEW

File: components/automate-cli/cmd/chef-automate/cleanup.go [hunk 1]
```diff
@@ -59,46 +75,46 @@ func runCleanupCmd(cmd *cobra.Command, args []string) error {
...
-					writer.Println("cleanup is starting on " + servername + " node : " + automateIps[i]+ "\n")
+					writer.Println("Cleanup has started on " + servername + " node : " + automateIps[i] + "\n")
 					_, err := ConnectAndExecuteCommandOnRemote(sshUser, sshPort, sskKeyFile, automateIps[i], FRONTENDCLEANUP_COMMANDS)
 					if err != nil {
 						writer.Errorf("%s", err.Error())
 						return err
 					} else {
-						writer.Success("cleanup is successfully completed on " + servername + " node : " + automateIps[i] + "\n")
+						writer.Success("Cleanup is completed on " + servername + " node : " + automateIps[i] + "\n")
 					}
```
Comment (punitmundra) [inline components/automate-cli/cmd/chef-automate/cleanup.go:85]: can we write this code in func

File: components/automate-cli/cmd/chef-automate/cleanup.go [hunk 2]
```diff
@@ -59,46 +75,46 @@ func runCleanupCmd(cmd *cobra.Command, args []string) error {
...
+					writer.Println("Cleanup has started on " + servername + " node : " + chefserverIps[i] + "\n")
 					_, err := ConnectAndExecuteCommandOnRemote(sshUser, sshPort, sskKeyFile, chefserverIps[i], FRONTENDCLEANUP_COMMANDS)
 					if err != nil {
 						writer.Error(err.Error())
 						return err
 					} else {
-						writer.Success("cleanup is successful on " + servername + " node : " + chefserverIps[i] + "\n")
+						writer.Success("Cleanup is completed on " + servername + " node : " + chefserverIps[i] + "\n")
 					}
 				}
```
Comment (punitmundra) [inline components/automate-cli/cmd/chef-automate/cleanup.go:97]: we can create func like backendpostgrescleanup()

File: components/automate-cli/cmd/chef-automate/cleanup.go [hunk 3]
```diff
@@ -111,6 +127,53 @@ func runCleanupCmd(cmd *cobra.Command, args []string) error {
...
+						"-c",
+						fmt.Sprintf(AWS_PROVISION, appendString),
+					}
+					err = executeCommand("/bin/sh", args, "")
+					if err != nil {
+						return err
+					}
+				}
+				cleanUpScript := "hab pkg uninstall chef/automate-ha-deployment"
+				args = []string{
```
Comment (punitmundra) [inline components/automate-cli/cmd/chef-automate/cleanup.go:167]: break the code in functions

File: components/automate-cli/cmd/chef-automate/cleanup.go (hunk 1 above)
Comment (ArvinthC3000) [reply components/automate-cli/cmd/chef-automate/cleanup.go]: Outdated

File: components/automate-cli/cmd/chef-automate/cleanup.go (hunk 2 above)
Comment (ArvinthC3000) [reply components/automate-cli/cmd/chef-automate/cleanup.go]: Outdated

**Pull request merged**